var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -36,16 +36,23 @@ class Qmcpack(CMakePackage):
     version('3.1.0', 'bdf3acd090557acdb6cab5ddbf7c7960')
     version('3.0.0', '75f9cf70e6cc6d8b7ff11a86340da43d')
 
-    # This download method is untrusted, and is not recommend by the Spack manual.
+    # This download method is untrusted, and is not recommended
+    # by the Spack manual.
     version('develop', git='https://github.com/QMCPACK/qmcpack.git')
 
     # These defaults match those in the QMCPACK manual
     variant('debug', default=False, description='Build debug version')
     variant('mpi', default=True, description='Build with MPI support')
-    variant('cuda', default=False, description='Enable CUDA and GPU acceleration.')
-    variant('complex', default=True, description='Build the complex (general twist/k-point) version')
-    variant('mixed', default=False, description='Build the mixed precision (mixture of single and double precision) version for gpu and cpu (experimental)')
-    variant('gui', default=False, description='Install with Py-Matplotlib (long installation time)') 
+    variant('cuda', default=False,
+            description='Enable CUDA and GPU acceleration.')
+    variant('complex', default=True,
+            description='Build the complex (general twist/k-point) version')
+    variant('mixed', default=False,
+            description='Build the mixed precision (mixture of single and '
+                        'double precision) version for gpu and cpu '
+                        '(experimental)')
+    variant('gui', default=False,
+            description='Install with Matplotlib (long installation time)')
```

Comment:
Not necessarily requesting a change, but I noticed that you changed all the variants from being on a single line to multiple lines (presumably to make sure that it is under 80 characters). I'm just saying that you don't need to do this as we ignore variants.